config/routes.rb

```diff
@@ -294,6 +294,16 @@
       end
     end
 
+    scope module: :ects, path: "/ects/:ect_id", as: :ects do
+      namespace :change_email_address_wizard, path: "change-email-address" do
```

Comment:
We should consolidate the change wizard namespaces into a single module scope. The first one is defined above this block.